tests/playwright/src/specs/container-smoke.spec.ts

```diff
@@ -101,7 +103,9 @@ describe('Verification of container creation workflow', async () => {
     const imageDetails = await images.openImageDetails(imageToPull);
     const runImage = await imageDetails.openRunImage();
     await pdRunner.screenshot('containers-run-image.png');
-    const containers = await runImage.startContainer(containerToRun);
+    await runImage.startContainer(containerToRun, containerStartParams);
+    const containers = new ContainersPage(page);
+    await playExpect(containers.header).toBeVisible();
```

Comment:
If the startContainer() returns void, why would this be valid? what is handling the navigation to ContainersPage in order to make this assertion consistent?